root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4984,6 +4989,55 @@ function doMatchSubtest(
   previousMatchTests.push(entityType + '+' + url);
 }
 
+function doOptionsSubtest(
```

Comment:
I suggest naming tests of `CLEANUPS.restrict` with a more closely related name:

```suggestion
function doRestrictSubtest(
```

(“Options” may also refer to relationship attributes.)